drivers/telescope/lx200ap_experimental.cpp

```diff
@@ -501,7 +567,7 @@ bool LX200AstroPhysicsExperimental::ISNewSwitch(const char *dev, const char *nam
         IDSetSwitch(&APGuideSpeedSP, nullptr);
         return true;
     }
-
+#ifdef no
```

Comment:
should we use a better identifier than **no** ?

Reply:
I used the #ifdef as a temporary indicator. I suggest, that all with #ifdef no labeled sections will be removed right before the driver will be renamed (loosing "experimental"). I recommend to leave it.